package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "isncsci",
-  "version": "2.0.4",
+  "version": "2.0.5",
```

Comment:
I increased the minor version so we can publish a new version of the Algorithm and its packages.